packages/experimental/src/components/Notifications/Notifications.js

```diff
@@ -0,0 +1,293 @@
+          />
+        )}
+        {notification.type === 'success' && (
+          <CheckmarkFilled16
+            className={[
+              `${prefix}-notifications-panel-notification-status-icon`,
+              `${prefix}-notifications-panel-notification-status-icon-success`,
+            ].join(' ')}
+          />
+        )}
+        {notification.type === 'warning' && (
+          <WarningAltFilled16
+            className={[
+              `${prefix}-notifications-panel-notification-status-icon`,
+              `${prefix}-notifications-panel-notification-status-icon-warning`,
+            ].join(' ')}
+          />
+        )}
+        {notification.type === 'informational' && (
+          <InformationSquareFilled16
+            className={[
+              `${prefix}-notifications-panel-notification-status-icon`,
+              `${prefix}-notifications-panel-notification-status-icon-informational`,
+            ].join(' ')}
+          />
+        )}
+        <div className={`${prefix}-notifications-panel-notification-content`}>
+          <p
+            className={`${prefix}-notifications-panel-notification-time-label`}>
+            {timeAgo(notification.timestamp)}
+          </p>
+          <h6 className={`${prefix}-notifications-panel-notification-title`}>
+            {notification.title}
+          </h6>
+          {notification.description &&
+            notification.description.length &&
+            renderDescription(notification.id)}
+          {notification.link &&
+            notification.link.text &&
+            notification.link.url && (
+              <Link
+                href={notification.link.url}
+                className={`${prefix}-notifications-panel-notifications-link`}>
+                {notification.link.text}
+              </Link>
+            )}
+        </div>
+      </div>
+    );
+  };
+
+  return (
+    shouldRender && (
+      <div
+        className={[`${prefix}-notifications-panel-container`].join(' ')}
+        style={{ animation: `${open ? 'fadeIn 250ms' : 'fadeOut 250ms'}` }}
+        onAnimationEnd={onAnimationEnd}>
+        <div className={`${prefix}-notifications-header-container`}>
+          <div className={`${prefix}-notifications-header-flex`}>
+            <h6 className={`${prefix}-notifications-header`}>Notifications</h6>
+            <Button
+              size="small"
+              kind="ghost"
+              className={`${prefix}-notifications-dismiss-button`}
+              onClick={() => {
+                setAllNotifications([]);
+              }}>
+              Dismiss all
+            </Button>
+          </div>
+          <ToggleSmall
+            className={`${prefix}-notifications-do-not-disturb-toggle`}
+            id={`${prefix}-notifications-do-not-disturn-toggle-component`}
+            labelA="Do not disturb"
+            labelB="Do not disturb"
+            onToggle={(event) => onDisableNotificationChange(event)}
+            aria-label="Do not disturb toggle"
```

Comment:
Is an aria-label needed when other labels are visible?

Reply:
Not 100% sure I just thought it was safest to add it just in case.